verification/curator-service/ui/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
Should probably `.ts` this, given that it's in src.

Reply:
Actually no, the docs say that this should be named setupProxy.js.
react automatically calls that if it exists, I am not sure we can make it a typescript thing considering app is being passed as a param and comes from "somewhere" that I couldn't find out.